--- origsrc/qt-gstreamer-0.10.3/cmake/modules/QtGStreamerConfigCommon.cmake	2013-10-14 03:44:40.000000000 -0500
+++ src/qt-gstreamer-0.10.3/cmake/modules/QtGStreamerConfigCommon.cmake	2013-12-24 19:04:02.608912400 -0600
@@ -25,6 +25,6 @@ if (CMAKE_COMPILER_IS_GNUCXX)
         if (NOT QTGSTREAMER_FLAGS) # be quiet if we try to find QtGStreamer multiple times
             message(STATUS "GCC 4.5 or later detected. Enabling C++0x support in QTGSTREAMER_FLAGS.")
         endif()
-        set(QTGSTREAMER_FLAGS "-std=c++0x")
+        set(QTGSTREAMER_FLAGS "-std=gnu++0x")
     endif()
 endif()
