--- origsrc/qt-gstreamer-0.10.2/CMakeLists.txt	2012-04-15 04:03:08.000000000 -0500
+++ src/qt-gstreamer-0.10.2/CMakeLists.txt	2013-09-03 18:18:09.793752300 -0500
@@ -109,6 +109,7 @@ if (CMAKE_COMPILER_IS_GNUCXX)
 endif ()
 
 set(LIB_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX} CACHE STRING "Destination directory for libraries")
+set(BIN_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/bin CACHE STRING "Destination directory for DLLs")
 set(INCLUDES_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/include/QtGStreamer)
 
 if (USE_GST_PLUGIN_DIR)
--- origsrc/qt-gstreamer-0.10.2/src/QGlib/CMakeLists.txt	2012-04-15 04:03:08.000000000 -0500
+++ src/qt-gstreamer-0.10.2/src/QGlib/CMakeLists.txt	2013-09-03 18:18:25.539653000 -0500
@@ -53,5 +53,8 @@ target_link_libraries(QtGLib ${QT_QTCORE
 target_link_libraries(QtGLib LINK_INTERFACE_LIBRARIES ${QT_QTCORE_LIBRARY})
 
 # Install
-install(TARGETS QtGLib DESTINATION ${LIB_INSTALL_DIR} EXPORT ${EXPORT_TARGET_SET})
+install(TARGETS QtGLib EXPORT ${EXPORT_TARGET_SET}
+                       LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+                       RUNTIME DESTINATION ${BIN_INSTALL_DIR}
+                       ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
 install_headers("QGlib" ${QtGLib_INSTALLED_HEADERS})
--- origsrc/qt-gstreamer-0.10.2/src/QGst/CMakeLists.txt	2012-04-15 04:03:08.000000000 -0500
+++ src/qt-gstreamer-0.10.2/src/QGst/CMakeLists.txt	2013-09-03 18:18:33.592113500 -0500
@@ -164,7 +164,9 @@ target_link_libraries(QtGStreamerUtils $
 target_link_libraries(QtGStreamerUtils LINK_INTERFACE_LIBRARIES ${QTGSTREAMER_LIBRARY})
 
 # Install
-install(TARGETS QtGStreamer DESTINATION ${LIB_INSTALL_DIR} EXPORT ${EXPORT_TARGET_SET})
-install(TARGETS QtGStreamerUi DESTINATION ${LIB_INSTALL_DIR} EXPORT ${EXPORT_TARGET_SET})
-install(TARGETS QtGStreamerUtils DESTINATION ${LIB_INSTALL_DIR} EXPORT ${EXPORT_TARGET_SET})
+install(TARGETS QtGStreamer QtGStreamerUi QtGStreamerUtils
+        EXPORT ${EXPORT_TARGET_SET}
+        LIBRARY DESTINATION ${LIB_INSTALL_DIR}
+        RUNTIME DESTINATION ${BIN_INSTALL_DIR}
+        ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
 install_headers("QGst" ${QtGStreamer_INSTALLED_HEADERS})
